Frends.Community.ActiveMQ.Tests/Frends.Community.ActiveMQ.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net471</TargetFrameworks>
+    <TargetFrameworks>net6</TargetFrameworks>

```

Comment:
I don't think it's a good idea to use target in tests which is not used in actual task. Unit tests should be run using same target that is used in the task. In this case, either framework or standard.